prow/tide/tide.go

```diff
@@ -41,8 +42,8 @@ import (
 
 const (
 	statusContext   string = "tide"
-	statusInPool           = "In tide pool."
-	statusNotInPool        = "Not in tide pool."
+	statusInPool           = "In merge pool."
+	statusNotInPool        = "Not mergeable.%s"
```

Comment:
Add a comment explaining the `%s`.

Reply:
Also, comment in what case we may not know %s if there is one

Reply:
Done.